src/fundus/publishers/fr/__init__.py

```diff
@@ -13,3 +15,17 @@ class FR(PublisherEnum):
         ],
         parser=LeMondeParser,
     )
+
+    Euronews = PublisherSpec(
+        name="Euronews",
+        domain="https://www.euronews.com/",
+        sources=[
+            Sitemap("https://www.euronews.com/sitemaps/en/articles.xml"),
+            Sitemap("https://de.euronews.com/sitemaps/de/articles.xml"),
+            Sitemap("https://fr.euronews.com/sitemaps/fr/articles.xml"),
+            NewsMap("https://www.euronews.com/sitemaps/en/latest-news.xml"),
+            NewsMap("https://de.euronews.com/sitemaps/de/latest-news.xml"),
+            NewsMap("https://fr.euronews.com/sitemaps/fr/latest-news.xml"),
+        ],
+        parser=EuronewsParser,
```

Comment:
Since this was added to `fr` could you rename the publisher to `EuronewsFR` and add only sources with `fr` subdomain?

If the page layout stays the same, it would be nice if you could add the other languages to their corresponding regions and reuse the parser. You wouldn't have to create three separate files, just declare the parser in one region and import it in the others. If not, don't worry, adding just this one is enough ;)

Reply:
For fr, en and de the layout is the same, the other languages I haven’t tried, yet. (I expect the same though). 
I’m not sure I’ll have time to add all the other languages that they offer but I’ll split fr, en and de tonight.

Reply:
@dkm1006 That looks really great! Thanks a lot :) 

Good idea. I added a `shared` directory to the publisher dir.